### PR TITLE
ti_misp: add toggle to enable request tracing

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Adds an advanced option to enable http request trace logging for debugging purposes and bumps kibana constraint to 8.5.0 which is when the request tracer feature was introduced to Agent.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- For #5956

### CODE REVIEW

File: packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs [hunk 1]
```diff
@@ -2,6 +2,9 @@ config_version: "2"
 interval: {{interval}}
 request.method: "POST"
 
+{{#if enable_request_tracer}}
+request.tracer.filename: http-request-trace-httpjson-akamai.ndjson
```
Comment (kgeller) [inline packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs:6]:
```suggestion
request.tracer.filename: http-request-trace-httpjson-ti_misp-threat.ndjson
```

File: packages/ti_misp/manifest.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+version: "1.13.0"
 release: ga
 description: Ingest threat intelligence indicators from MISP platform with Elastic Agent.
 type: integration
 format_version: 1.0.0
 license: basic
 categories: ["security", "threat_intel"]
 conditions:
-  kibana.version: ^8.0.0
+  kibana.version: ^8.5.0
```
Comment (kcreddy) [inline packages/ti_misp/manifest.yml:11]: I see for `ti_misp`, some kibana dashboard versions are at `8.0.0`. With this change, does it effect dashboard loading?

Comment (efd6) [reply packages/ti_misp/manifest.yml:11]: This won't affect those.

**Review by kcreddy — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**